assets/dev/scss/editor/_select2.scss

```diff
@@ -47,6 +47,18 @@
 			border-color: $editor-light;
 		}
 	}
+
+	&:not(.select2-container--all-selected) .select2-selection--multiple .select2-search--inline::before {
```

Comment:
Add to the `.select2-search--inline` class a `padding-top: 0.6px` to place it in the accurate location